themes/mcf/pages/about.tsx

```diff
@@ -18,24 +18,24 @@ const About = () => {
       <section className="pt-8">
         <SiteWidth>
           <SingleCol>
-            <div className="text-content mb-12">
-              <Heading level={1} extraClasses="custom-header">
+            <div>
+              <Heading level={1} extraClasses="custom-header mb-4">
                 About
               </Heading>
-              <p>
+              <p className="text-content mb-12">
```

Comment:
Same comment as below, put `text-content` on wrapping div on line 21 rather than repeating throughout

Reply:
As below discussed and makes sense!